### ISSUE TITLE
Pre-train for Episode Dialog Summary Embeddings

### ISSUE THREAD

**Issue by Lee-Ft** [InterDigitalInc/DialogSummary-VideoQA#1]
Hi,
  My GPU (2080Ti *4) has no sufficient memory for episode dialog summary pre-training. I tried to change some hyperparameters to fit into experiments. However, the accuracy was a little bit worse than reported in paper.
Here is my parameter settings and training results:
`python stream_main_train.py --train_name episode_dialog_summary --max_seq_length 200 --seq_stride 100 --mini_batch_size 2`

```
12/01/2021 06:14:14 - INFO - utils -   Total: 0.684
12/01/2021 06:14:14 - INFO - utils -   Visual : 0.635
12/01/2021 06:14:14 - INFO - utils -   Textual : 0.598
12/01/2021 06:14:14 - INFO - utils -   Temporal : 0.698
12/01/2021 06:14:14 - INFO - utils -   Knowledge : 0.716
```

So, I have two questions, 
- How does `max_seq_length` and `seq_stride` influence the results? If I have to change one of them (or both), which one would not be such harmful?
- Is there any plan to release the pre-trained embeddings?

Thanks a lot!

**Comment by engindeniz** [comment]
Hi, 

Thank you for your interest.

- Since long input sources are longer than the transformer’s maximum sequence length, they are split into overlapping parts in a sliding window fashion. `max_seq_length `refers to the length of one part, and `seq_stride `means the length of stride. They are both critical hyperparameters that may cause performance degradation. You can check the episode input sources section in the paper for more details.
- I have shared our main embeddings and stream models via your email.

**Comment by deniz-engin** [comment]
Hi, 

Thank you for your interest.

- Since long input sources are longer than the transformer’s maximum sequence length, they are split into overlapping parts in a sliding window fashion. `max_seq_length `refers to the length of one part, and `seq_stride `means the length of stride. They are both critical hyperparameters that may cause performance degradation. You can check the episode input sources section in the paper for more details.
- I have shared our main embeddings and stream models via your email.
